src/components/menu/menu.types.ts

```diff
@@ -1,10 +1,50 @@
-import { MenuListItem } from '../menu-list/menu-list-item.types';
+import { ListSeparator } from '@limetech/lime-elements';
 
 export type OpenDirection = 'left' | 'right';
 
-export interface MenuItem<T = any> extends MenuListItem<T> {
+export interface MenuItem<T = any> {
     /**
      * The additional supporting text is used for shortcut commands and displayed in the list item.
      */
     commandText?: string;
+
+    /**
+     * Text to display in the list item.
```

Comment:
Shouldn't we change all list item to _menu item_ here?

Reply:
True! Good catch 👍